class.jetpack.php

```diff
@@ -5651,18 +5651,18 @@ public function alert_identity_crisis() {
 				</div>
 
 				<div class="jp-id-crisis-question" id="jp-id-crisis-question-2" style="display: none;">
-					<p><?php printf( __( 'Are  <strong> %2$s </strong> and <strong> %1$s </strong> two completely separate websites? If so we should create a new connection, which will reset your followers and linked services. <a href="#" title="What does resetting the connection mean?"><em>What does this mean?</em></a>', 'jetpack' ), $errors[ $key ], (string) get_option( $key ) ); ?>
+					<p><?php printf( __( 'Are  <strong> %2$s </strong> and <strong> %1$s </strong> two completely separate websites? If so we should create a new connection, which will reset your followers and linked services. %3s What does this mean? %4s', 'jetpack' ), $errors[ $key ], (string) get_option( $key ), "<a href='https://jetpack.me/support/what-does-resetting-the-connection-mean/' target='_blank' title='What does resetting the connection mean?'><em>", "</em></a>" ); ?>
 					</p>
 					<div class="btn-group">
 						<a href="#" class="button reset-connection">Reset the connection</a>
 						<a href="#" class="button is-dev-env">This is a development environment</a>
-						<a href="https://jetpack.me/support" class="button contact-support">Submit a support ticket</a>
+						<a href="https://jetpack.me/contact-support/" class="button contact-support">Submit a support ticket</a>
```

Comment:
This string and the ones directly above it will need to be translatable... guess we missed this in the prior PR